spec/decorators/provider_decorator_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe ProviderDecorator do
+  describe "#formatted_address" do
+    it "returns a formatted address" do
```

Comment:
So we don't have to test different scenarios in the systems specs, it might be nice to have a unit spec where there isn't the full complement of address detail (eg only one line of the address, and no county).

Reply:
resolved in [Refactor importer to align with GiasImporter](https://github.com/DFE-Digital/itt-mentor-services/pull/92/commits/95894fbc58176adec76570e9578c60a49761f274)